.gitignore

```diff
@@ -3,3 +3,15 @@
 dist/
 .DS_Store
 **/.DS_Store
+
+# VSCode
+bin/
+.vscode
+.settings
+.project
+.classpath
+.factorypath
+
+# IntelliJ
+.idea
+*.iml
```

Comment:
I'm a little bit against adding editors exclusion to committed github files, just because we need to do that in each editor and each user might have different files to ignore. What about using a [global ignore file](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer)?